apps/src/required_block_utils.js

```diff
@@ -1,5 +1,6 @@
 /* global Text */
 
+var experiments = require('@cdo/apps/util/experiments');
```

Comment:
Can we switch based on which wrapper we are using, rather than which experiment?